### DISCUSSION TITLE
Get the HTML video element from YouTube component

### DISCUSSION THREAD

**Discussion by paolodina** [spences10/sveltekit-embed#487] [Q&A] [answered]
Hi @spences10 and all, I'm test driving this package and works great so far. I'm using the YouTube embed and the question is, is there a way to have at least the html video element? I need it to pause, play, and generally have the whole video api at my disposal.

It would also be handy to have events (like onReady, etc). Looking at the source code I didn't see something like that. I need that feature and I can work on it and eventually submit a PR. Is it something the project would be interested in?  Suggestions, opinions etc greatly appreciated.

**Edit.** nevermind, I realized it's out of the scope of this package.

**Comment by spences10** [comment, marked as answer]
Hey @paolodina sounds great, like I think you twigged onto though that what you're suggesting may in fact be it's own package.

There's some additional options I can add, I've made an issue for it #488
